tasks/comment/InfracostComment/index.ts

```diff
@@ -12,12 +11,16 @@ import { promises as fsPromises } from 'fs';
  * @param behavior - post behavior, used by `infracost output` command
  * @return the saved file path
  */
-function generateOutput(path: string, format: string, behavior: string): string {
-  const outputFile = '/tmp/infracost-comment.md';
```

Comment:
Figured it won't work on windows, guess it's alright to output to the current directory.